CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 
 All functional changes to the project are documented in this file.
 
+## [2025.1] (24 Feb 2025)
+
+- gh-163: drop `Python 3.9` & `NumPy < 2.1.1` support (#391)
+- gh-368: add type overloading for `lensing.from_convergence` (#395)
+- gh-384: rename all internal functions to `trapezoid` over `trapz` (#392)
+- gh-440: remove scipy as a test dependency (#462)
```

Comment:
https://github.com/glass-dev/glass/pull/444 was a bug fix too, should it be included?

Reply:
Yep, good spot. WIll add now.